test/volumes-basic.test.js

```diff
@@ -326,7 +326,8 @@ if (CONFIG.experimental_nfs_shared_volumes !== true) {
     });
 
     test('teardown', function (t) {
-        common.teardown(CLIENTS, SERVER, function () {
+        common.teardown(CLIENTS, SERVER, function (err) {
+            t.ifErr(err, 'teardown should be successfull, got: ' + err);
```

Comment:
English sucks. But it's `successful` not `successfull`. I bet nobody can explain why.